util/types/time.go

```diff
@@ -410,7 +348,9 @@ func (t Time) roundFrac(fsp int) (Time, error) {
 		return t, nil
 	}
 
-	nt := t.Time.GoTime().Round(gotime.Duration(math.Pow10(9-fsp)) * gotime.Nanosecond)
+	t1, _ := t.Time.GoTime()
+	// TODO: Fix here.
```

Comment:
It would return error, but I don't know how to handle the error currently, so add a TODO.

It time contain 00, after convert GoTime(), we'll get the nearest time to that date, for example:
2016-12-00 -> 2016-11-30 23:59:59